source/sass/patterns/molecules/tag-list.scss

```diff
@@ -23,23 +23,7 @@
 .tag-list__item {
   display: inline-block;
   @include label-tag-typography();
-
-  &:after {
-    display: inline;
-    content: ",\a0";
-  }
-
-  &:lang(ar):after {
-    content: "،\a0";
-  }
-
-  &:lang(ja):after {
-    content: "、";
-  }
-
-  &:last-child:after {
-    content: "";
-  }
+  @include comma-separator();
```

Comment:
Unfortunately CLDR doesn't provide alternative separation (for affiliations we should use a semi-colon in English, as the items probably already contain a comma).

There's a brief mention at http://cldr.unicode.org/development/development-process/design-proposals/list-formatting, but no links to any discussions.